labop/lib/sample_arrays.py

```diff
@@ -123,6 +123,20 @@
 p.add_input('mount', 'http://bioprotocols.org/uml#ValueSpecification', optional=True)
 doc.add(p)
 
+p = labop.Primitive('JoinMetadata')
+p.description = 'Associate a labop:SampleMetadata with a labop.Dataset to create a labop.Dataset'
+p.add_input('metadata', 'http://bioprotocols.org/labop#SampleMetadata')
+p.add_input('data', 'http://bioprotocols.org/labop#Dataset'),
```

Comment:
Since it's datasets in and out, I think the names want to change, e.g., to "dataset" and "enhanced_dataset"

Reply:
you got it.